lms/djangoapps/course_api/views.py

```diff
@@ -170,4 +175,10 @@ def get_queryset(self):
         Return a list of courses visible to the user.
         """
         username = self.request.query_params.get('username', self.request.user.username)
-        return list_courses(self.request, username)
+        courses = list_courses(self.request, username)
+
+        org = self.request.query_params.get('org')
```

Comment:
Can we move this to within `list_courses` in api.py?  It seems like a generally useful filter that can be used by internal callers as well.